includes/specials/SpecialListgrouprights.php

```diff
@@ -60,7 +60,10 @@ public function execute( $par ) {
 				'</tr>'
 		);
 
-		$allGroups = User::getAllGroups();
+		$allGroups = array_unique( array_merge(
+			User::getAllGroups(),
```

Comment:
`getAllGroups` suggests it returns all groups and not (I presume) "explicit" only
or I'm missing something?